zonemaster/zonemaster-gui
# Update dependencies with non-breaking changes

## Purpose

This PR updates dependencies with non-breaking changes using `npm audit fix`.

## Context

Using the [latest GUI version (4.3.0)](https://github.com/zonemaster/zonemaster-gui/tree/v4.3.0), running `npm install` shows the following:
`25 vulnerabilities (3 low, 4 moderate, 15 high, 3 critical)`

## Changes

Updating dependencies with non-breaking changes resolve 12 vulnerabilities and now shows:
`13 vulnerabilities (1 moderate, 10 high, 2 critical)`

## How to test this PR

From this PR, run `npm install` and then `npm run release` to produce the zip file. There should be no errors. Then install GUI from that file, and check that it works properly.

I'll also attach the updated zip file here: [zonemaster_web_gui.zip](https://github.com/user-attachments/files/18024003/zonemaster_web_gui.zip)
